attributes/config.rb

```diff
@@ -1,6 +1,6 @@
 default['mapr']['config']['owner'] = 'mapr'
 default['mapr']['config']['group'] = 'mapr'
-default['mapr']['config']['mode'] = 0o600
+default['mapr']['config']['mode'] = 0o644
```

Comment:
This attribute is used in recipe/security.rb recipe please update  it first (the recipe), before setting it here